eclass/distutils-r1.eclass

```diff
@@ -628,11 +628,11 @@ distutils_enable_tests() {
 			;;
 		unittest)
 			# unittest-or-fail is needed in py<3.12
-			test_deps+="
-				$(python_gen_cond_dep '
+			local test_pkgs="$(python_gen_cond_dep '
```

Comment:
I don't think you need `"` quoting here (since direct `$()` assign into var). But maybe safer to include them in case someone modifies in future...